engine.py

```diff
@@ -249,7 +274,6 @@ def __getattr__(self, name):
                                      "run UI applications from Houdini.")
 
         base = {"qt_core": QTProxy(), "qt_gui": QTProxy(), "dialog_base": None}
-        self._ui_type = None
```

Comment:
I had moved the initialization of `_ui_type` up into `init_engine` so I think it should always be defined.